les1.js

```diff
@@ -0,0 +1,8 @@
+const fn1 = (sayHello) => {
+    if (sayHello === "Mark") {
```

Comment:
what if I call your function with lower cased mark? when you compare strings, and you want be sure about their equality you have to compare them in one register